types/experiments.ts

```diff
@@ -0,0 +1,81 @@
+/** All fields are publically accessible via the API and available to clients */
+interface Experiment {
```

Comment:
Q: Is there somewhere that we're specifying the "Experiment Owner" from the UI mockup?
https://www.figma.com/file/5SLa07XqQb0ttB1ay1gami/WIP-Platform-Exp?node-id=214583%3A0

Reply:
I don't think the owner is necessary on the public api, although we definitely want to continue to store it